runtime-sdk/src/testing/mock.rs

```diff
@@ -232,14 +236,45 @@ impl Signer {
         C: Context,
         B: cbor::Encode,
     {
+        let inner_call = transaction::Call {
```

Comment:
No need to create `inner_call`, if you move inside `if` block.

Reply:
The value of `inner_call` is used in both branches of the `if`, in one it's wrapped and in the other it's just moved to a new variable, I wanted to avoid repeating the constructor. I restructured the if slightly, maybe it's a bit more readable now.